agent/lib/PbenchBase.pm

```diff
@@ -171,8 +171,9 @@ sub load_benchmark_config {
 
 sub metadata_log_begin_run {
     my $benchmark_run_dir = shift;
+    my $benchmark_name = shift;
     my $group = shift;
-    system("pbench-metadata-log --group=" . $group . " --dir=" . $benchmark_run_dir . " beg");
+    system("benchmark=" . $benchmark_name . " pbench-metadata-log --group=" . $group . " --dir=" . $benchmark_run_dir . " end");
```

Comment:
Are we sure this is correct?  The subroutine is `metadata_log_begin_run` so using `beg` as the argument seems more logical than `end` to me, but I can't profess to fully understanding the metadata log usage/behavior.

Reply:
Yes, copy-paste-error on my part.  Thanks for catching that!